python/pyspark/ml/tuning.py

```diff
@@ -545,6 +551,10 @@ class _TrainValidationSplitParams(_ValidatorParams):
     trainRatio = Param(Params._dummy(), "trainRatio", "Param for ratio between train and\
      validation data. Must be between 0 and 1.", typeConverter=TypeConverters.toFloat)
 
+    def __init__(self, *args):
```

Comment:
You added `*args` many places. Just to make sure it is for consistency?

Reply:
Yes, it's for consistency.